### PR TITLE
feat(Tools/DbImport): implement separated app for importing DB

### PR DESCRIPTION
**Pull request by Winfidonarleyan**
## Changes Proposed:
- Add new tool `dbimport` for import all `.sql` files in database
- Added support `pending` sql's for all builds in CI

## Tested:
- Tested with CI and sql file from PR #11922

### CODE REVIEW

**Comment by Kitzunu** [comment]
Not apps. Tools

**Comment by Winfidonarleyan** [comment]
> Not apps. Tools

Database lib is server dep

**Comment by Kitzunu** [comment]
can we export it somehow? Because this is just a tool, like the map extractors

**Comment by Winfidonarleyan** [comment]
> can we export it somehow? Because this is just a tool, like the map extractors

maybe

**Comment by Winfidonarleyan** [comment]
@Kitzunu done, in tools :D

**Comment by Kitzunu** [comment]
Pog. But build fails

File: src/tools/dbimport/dbimport.conf.dist [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+###################################################################################################
+
+###################################################################################################
+# DB IMPORT CONFIG
+#
+#    LogsDir
+#        Description: Logs directory setting.
+#        Important:   LogsDir needs to be quoted, as the string might contain space characters.
+#                     Logs directory must exists, or log file creation will be disabled.
+#        Example:     "/home/youruser/warheadcore/logs"
```
Comment (IntelligentQuantum) [inline src/tools/dbimport/dbimport.conf.dist:38]: Maybe azerothcore

File: src/tools/dbimport/dbimport.conf.dist [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+#        Default:     "" - (Log files will be stored in the current path)
+
+LogsDir = ""
+
+#
+#    SourceDirectory
+#        Description: The path to your WarheadCore source directory.
```
Comment (IntelligentQuantum) [inline src/tools/dbimport/dbimport.conf.dist:45]: here

File: src/tools/dbimport/dbimport.conf.dist [hunk 3]
```diff
@@ -0,0 +1,286 @@
...
+#                     If the path is left empty, the built-in CMAKE_SOURCE_DIR is used.
+#        Example:     "../WarheadCore"
```
Comment (IntelligentQuantum) [inline src/tools/dbimport/dbimport.conf.dist:47]: here

**Review by IntelligentQuantum — APPROVED** [review]

**Comment by Winfidonarleyan** [comment]
@Kitzunu can check again?

File: apps/ci/ci-conf-tools.sh [hunk 4]
```diff
@@ -7,7 +7,6 @@ MTHREADS=$(($(grep -c ^processor /proc/cpuinfo) + 2))
 CWARNINGS=ON
 CDEBUG=OFF
 CTYPE=Release
-CSCRIPTS=static
```
Comment (acidmanifesto) [inline apps/ci/ci-conf-tools.sh:10]: Why remove the scripts?

File: modules/CMakeLists.txt [hunk 5]
```diff
@@ -290,7 +288,7 @@ target_include_directories(modules
     ${CMAKE_CURRENT_SOURCE_DIR}
     ${PUBLIC_INCLUDES})
 
-set_target_properties(scripts
```
Comment (acidmanifesto) [inline modules/CMakeLists.txt:293]: Why are we renaming?

**Comment by acidmanifesto** [comment]
https://github.com/azerothcore/azerothcore-wotlk/runs/6775872366?check_suite_focus=true#step:4:373
To
https://github.com/azerothcore/azerothcore-wotlk/runs/6775872366?check_suite_focus=true#step:4:419

Shows errors on windows build.

File: apps/ci/ci-conf-tools.sh (hunk 4 above)
Comment (Winfidonarleyan) [reply apps/ci/ci-conf-tools.sh:10]: this default

File: modules/CMakeLists.txt (hunk 5 above)
Comment (Winfidonarleyan) [reply modules/CMakeLists.txt:293]: this `modules` forlder

**Comment by Winfidonarleyan** [comment]
@acidmanifesto tools for win work now

**Comment by Nyeriah** [comment]
Thanks for looking into this 👍

**Pull request merged**